static/src/stylesheets/module/atoms/_snippets.scss

```diff
@@ -82,3 +82,24 @@
         }
     }
 }
+
+.explainer-snippet__credit {
```

Comment:
do we need this in the amp stylesheet as well?

Reply:
Yes!